packages/medusa/src/services/customer-group.ts

```diff
@@ -64,6 +64,14 @@ class CustomerGroupService extends BaseService {
       }
     })
   }
+
+  async list(groups: DeepPartial<CustomerGroup>[]): Promise<CustomerGroup[]> {
+    const cgRepo: CustomerGroupRepository = this.manager_.getCustomRepository(
+      this.customerGroupRepository_
+    )
+
+    return await cgRepo.findByIds(groups.map((g) => g.id))
+  }
```

Comment:
The `list` function should take a `FilterableCustomerGroupProps` as the first arg, and `FindConfig<CustomerGroup>` as the second arg.

The `BaseService` has a function called `buildQuery_` which will take these two parameters and convert them into a TypeORM query that can then be passed to `cgRepo.find()`.

See the implementation here for reference: https://github.com/medusajs/medusa/blob/master/packages/medusa/src/services/product-tag.ts#L88

The structure of `FilterableCustomerGroupsProps` should resemble this: https://github.com/medusajs/medusa/blob/master/packages/medusa/src/types/product.ts#L12

Note that many of the filterable props accept an array this is to make it possible to do `IN` queries. So in the case you are solving for here where you have a list of ids to get the correct list call would end up becoming:
```
customerGroupService.list({ id: [id1, id2, ....] })
```